src/config.ts

```diff
@@ -0,0 +1,18 @@
+import fs from 'fs';
+
+export interface Config {
+    labelsToSearch: string[];
+    numberOfReposPerCall: number;
+    inputFilePath: string;
+    outputFilePath: string;
+}
+
+export function readConfigFile(filePath: string): Config {
+    const configFilePath = `${__dirname}/../../${filePath}`;
```

Comment:
Here too, I think we should trust whatever path comes in, and not adjust it.

Reply:
Done